### DISCUSSION TITLE
Downloaded version requires macOS 10.15 or later

### DISCUSSION THREAD

**Discussion by Andreaux** [GrandOrgue/grandorgue#789] [Help]
Hello,

I am running macOS 10.14 Mojave and the downloaded build can't be run as it requires version 10.15 or later. Would it be possible to have a build that would run on earlier versions?
I'm not able to build from source code unfortunately, so I can't give that a try.

Many of us won't update to 10.15 because software we own (many plug-ins that were not updated by their developers) is still 32-bit and 32-bit software won't run on 10.15 and later.

I hope someone will have a solution.

Thanks,

Gabriel

**Comment by midimusic** [comment]
I was absolutely disgusted when Apple ruined my iMac by no longer supporting 32 bit applications so I stopped using it.

Later (using Bootcamp) I found that I could just as well run Windows and/or Linux on it and regain all the support I needed. GO will run perfectly well on either.

Even later I completely removed MacOS by using the whole disk for Linux and have never looked back (so far). The main problem? is that the screen resolution is much too high for some apps and this is sometimes a disadvantage.

csw900

**Comment by oleg68** [comment]
> Would it be possible to have a build that would run on earlier versions?

Unfortinally, it is not possible because GitHub doesn't provide an environment for building GO for earlier version of Mac.
